examples/step-27/doc/results.dox

```diff
@@ -209,7 +209,71 @@ include estimating the smoothness not on single cells, but cell assemblies or
 patches surrounding each cell. It may also be possible to find simple
 correction factors for each cell depending on the number of constrained
 degrees of freedom it has. In either case, there are ample opportunities for
-further research on finding good $hp$ refinement criteria. On the other hand,
-the main point of the current program was to demonstrate using the $hp$
-technology in deal.II, which is unaffected by our use of a possible
+further research on finding good $hp$-refinement criteria. On the other hand,
+the main point of the current program was to demonstrate using the
+$hp$-technology in deal.II, which is unaffected by our use of a possible
 sub-optimal refinement criterion.
+
+
+
+<a name="extensions"></a>
+<h3>Possibilities for extensions</h3>
+
+<h4>Different hp-decision strategies</h4>
+
+This tutorial demonstrates only one particular strategy to decide between h- and
+p-adaptation. In fact, there are many more ways to automatically decide on the
+adaptation type from which a couple are already implemented in deal.II:
+<ul>
+  <li><i>Fourier coefficient decay:</i> This is the strategy currently
+  implemented in this tutorial. For more information on this strategy, see
+  the general documentation of the SmoothnessEstimator::Fourier namespace.</li>
+
+  <li><i>Legendre coefficient decay:</i> This strategy is quite similar
+  to the current one, but uses Legendre series expansion rather than the
+  Fourier one: instead of sinusoids as basis functions, this strategy uses
+  Legendre polynomials. The transition from the Fourier strategy to
```

Comment:
```suggestion
  Legendre polynomials. Of course, since we approximate the solution using a finite-dimensional
  polynomial on each cell, the expansion of the solution in Legendre polynomials is also finite
  and, consequently, when we talk about the "decay" of this expansion, we can only consider the
  finitely many nonzero coefficients of this expansion, rather than think about it in asymptotic terms.
  But, if we have enough of these coefficients, we can certainly think of the decay of these
  coefficients as characteristic of the decay of the coefficients of the exact solution (which is, in
  general, not polynomial and so will have an infinite Legendre expansion), and considering the
  coefficients we have should reveal something about the properties of the exact solution.
  
  The transition from the Fourier strategy to
```